mep/accounts/partial_date.py

```diff
@@ -27,6 +27,12 @@ def from_db_value(self, value, expression, connection, context):
         using :meth:`to_python`'''
         return self.to_python(value)
 
+    def value_to_string(self, obj):
+        '''Customize string value for JSON serialization will'''
```

Comment:
my brain is having trouble with this docstring...it's not a sentence, right?

Reply:
😆 wow, nope. I think just the `will` is extra - does it make sense without that? I went through a couple versions of trying to describe this and apparently did not finish editing.